project/Common.scala

```diff
@@ -39,6 +39,23 @@ object Common extends AutoPlugin {
       "-Xfuture",
       "-target:jvm-1.8"
     ),
+    Compile / doc / scalacOptions := scalacOptions.value ++ Seq(
+      "-doc-title",
+      "Alpakka",
+      "-doc-version",
+      version.value,
+      "-sourcepath",
+      (baseDirectory in ThisBuild).value.toString,
+      "-doc-source-url", {
+        val branch = if (isSnapshot.value) "master" else s"v$version"
+        s"https://github.com/akka/alpakka/tree/${branch}€{FILE_PATH}.scala#L1"
```

Comment:
It's a pity this is restricted to Scala sources, now that Scaladoc generates docs even for Java code.
This will produce dead links for the things implemented in Java, right?

Reply:
See https://github.com/scala/bug/issues/5388

Reply:
Created #1378